### PR TITLE
Support UniDic format for MeCabTokenizer

### PR DESCRIPTION
**Pull request by altescy**
Thank you for this awesome library! 🙏 
Purpose of this PR is to use MeCabTokenizer with [UniDic format](https://unidic.ninjal.ac.jp/faq) dictionaries.

### CODE REVIEW

File: konoha/word_tokenizers/mecab_tokenizer.py [hunk 1]
```diff
@@ -84,12 +121,16 @@ def __init__(
         if dictionary_format is None:
             if system_dictionary_path is None or "ipadic" in system_dictionary_path.lower():
                 self._parse_feature = parse_feature_for_ipadic
+            elif system_dictionary_path is None or "unidic" in system_dictionary_path.lower():
```
Comment (himkt) [inline konoha/word_tokenizers/mecab_tokenizer.py:124]:
```suggestion
            elif "unidic" in system_dictionary_path.lower():
```

If `system_dictionary_path` is `None`, konoha assumes users attempt to create a tokenizer with `mecab-ipadic`. Therefore, it is not needed to check if `system_dictionary_path` is empty for `unidic`.

**Review by himkt — COMMENTED** [review]
Thank you @altescy, I'm happy to review this awesome PR!
I left the minor comments.

And could you please remove the line in https://github.com/himkt/konoha/pull/132/files#diff-0099b6884201201e9e578ae10139355c09edb9774b030db2a0703033e109bc62R120 (cuz konoha will support unidic in this PR)?

**Comment by altescy** [comment]
Thank you for your review!
I removed the test of `parse_feature_for_unidic` and modified the condition of dictionary format inference.

**Comment by himkt** [comment]
@altescy Please revert 4986db7, this test is helpful. :pray:
Let me merge after restoring the test.

**Comment by altescy** [comment]
@himkt I restored the test.

**Review by himkt — APPROVED** [review]
Again, thank you for your contribution!

**Pull request merged**